1.0/docs/start/first-element/step-5.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 5: Theming with custom CSS properties"
+subtitle: Build your first Polymer element
+---
+
+
+You now have a button that's basically functional. But it's stuck using the
+existing text color for both pressed and unpressed states. What if you want to
+get a little flashier?
+
+Local DOM helps prevent users from styling your element's internals by
+accident. Using custom properties, your element can expose a specific set of
+user-styleable properties.
+
+You apply a custom property inside your element using the `var` function.
+
+
+<pre class="prettyprint">
+ background-color: var(<em>--my-custom-property</em>, <em>defaultValue</em>);
+</pre>
+
+Where <code>--<em>my-custom-property</em></code> is a custom property name, always starting with two dashes (`--`), and <em>defaultValue</em> is an (optional) CSS value that's used if the custom property isn't set.
+
+Edit your element's `<style>` tag and replace the existing `fill` and `stroke` values with custom properties:
+
+#### icon-toggle.html
+{: .caption }
+
+
+<pre class="prettyprint">
+  &lt;style>
+    /* local styles go here */
+    :host {
+      display: inline-block;
+    }
+    iron-icon {
+      fill: var(--icon-toggle-color, rgba(0,0,0,0));
+      stroke: var(--icon-toggle-outline-color, currentcolor);
+    }
+    :host([pressed]) iron-icon {
+      fill: var(--icon-toggle-pressed-color, currentcolor);
+    }
+  &lt;/style>
+</pre>
+
+Because of the default values, you can still style the `<icon-toggle>` just by setting `color`, but now you have other options. Open up `demo/icon-toggle-demo.html `and set the new properties:
+
+#### icon-toggle-demo.html
+{: .caption }
+
+
+<pre class="prettyprint">
+    &lt;style>
+      :host {
+        font-family: sans-serif;
+        --icon-toggle-color: lightgrey;
+        --icon-toggle-outline-color: black;
+        --icon-toggle-pressed-color: red;
+      };
+    &lt;/style>
+</pre>
+
+Run the demo again to get colorful.
+
+
+
```

Comment:
I think it would be good to show how you can use the custom properties at a different level as well.

For instance, putting them in `demo/index.html` inside of a custom style tag (or linked stylesheet). Also showing how you have to reference an element to give the custom properties a scope. You can't put them under `body` for instance, they have to at least be under `icon-toggle-demo`. This is a really confusing part of the custom properties shim that lots of folks run into